cmd/plugins/balloons/policy/balloons-policy.go

```diff
@@ -1202,6 +1209,12 @@ func (p *balloons) fillCloseToDevices(blnDefs []*BalloonDef) {
 		if blnDef.PreferIsolCpus {
 			blnDef.PreferCloseToDevices = append(blnDef.PreferCloseToDevices, virtDevIsolatedCpus)
 		}
+		if blnDef.CoreType == "performance" {
+			blnDef.PreferCloseToDevices = append(blnDef.PreferCloseToDevices, virtDevPCores)
+		}
+		if blnDef.CoreType == "efficient" {
```

Comment:
ubernit: maybe it would have been warranted to use consts for "performance" and "efficient", since they appear to be referenced/used multiple times in the code.

Reply:
we have only one instance of "efficient" and "performance". So, I don't think we need a const here.